daemon/src/main/java/org/mvndaemon/mvnd/daemon/Server.java

```diff
@@ -270,7 +279,7 @@ private void client(SocketChannel socket) {
                 updateState(DaemonState.Idle);
                 return;
             }
-            LOGGER.info("Request received: " + message);
+            LOGGER.info("Request received: {}", message);
```

Comment:
Forgot about this one, it is technically not related to the issue, but easy enough fix so I bundled it here (it is even the same class, so hopefully that's ok).